File: listviewtutorial/app/src/main/java/com/example/listviewtutorial/MainActivity.java
package com.example.listviewtutorial;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.ArrayAdapter;
import android.widget.ListView;

public class MainActivity extends AppCompatActivity {
    String[] names=new String[]{"Aman","Himanshu","Vishal" };
    ListView lvnames;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        lvnames=findViewById(R.id.lvNames);
        ArrayAdapter<String> newAdapter=new ArrayAdapter<>(this,android.R.layout.simple_list_item_1,android.R.id.text1,
                                                                      names);
        lvnames.setAdapter(newAdapter);

    }
}

File: Recyclerview2/app/src/main/java/com/example/recyclerview/MainActivity.kt
package com.example.recyclerview

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
       buttonRecyclerView.setOnClickListener{
           val intent=Intent(this,HobbiesActivity::class.java)
           startActivity(intent)
       }
    }
}